content/_data/changelogs/weekly.yml

```
@@ -15089,6 +15089,69 @@
   # pull: 6433 (PR title: Fix typo in dragdrop-debug.js)
   # pull: 6436 (PR title: Bump junit from 1.57 to 1.58)
 
+  - version: '2.343'
+    date: 2022-04-08
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 4579
+        issue: 50211
+        authors:
+          - stellargo
+          - basil
+        pr_title: "[JENKINS-50211] Reject connections from agents with unsupported\
+          \ Remoting versions"
+        message: |-
+          Reject connections from agents with unsupported Remoting versions.
+      - type: rfe
+        category: rfe
+        pull: 5459
+        issue: 40700
+        authors:
+          - Aki-7
+          - oleg-nenashev
+          - basil
+        pr_title: "[FIX JENKINS-40700] Display communication protocol in agent logs"
+        message: |-
+          Display communication protocol in agent logs
```

Needs a full stop at the end of the line
```suggestion
          Display communication protocol in agent logs.
```